shlib/shcrypto/keys.go

```diff
@@ -143,10 +143,7 @@ func ComputeEpochSecretKeyShare(eonSecretKeyShare *EonSecretKeyShare, epochID *E
 
 // ComputeEpochID computes the id of the given epoch.
 func ComputeEpochID(epochIDBytes []byte) *EpochID {
-	epochIDBytesHash := keccak256(epochIDBytes)
-	epochIDBig := new(big.Int).SetBytes(epochIDBytesHash)
-	epochIDPoint := EpochID(*new(bn256.G1).ScalarBaseMult(epochIDBig))
-	return &epochIDPoint
+	return (*EpochID)(Hash1(epochIDBytes))
```

Comment:
Can you explain what's happing here? Is this just a type-cast to a `*EpochID` type?
This works because the EpochID is just a []byte type with additional methods, right?
(Asking out of curiosity)